chain/mainnet/mezo_31612-1/genesis.json

```diff
@@ -1 +1,296 @@
-[]
+{
+  "app_name": "mezod",
+  "app_version": "1.0.0-rc2",
+  "genesis_time": "2025-04-14T08:50:53.767481Z",
```

Comment:
Not that it matters much, but you may want to change this date to one closer to the launch date?

Reply:
We will modify this file and re-genesis for real mainnet so this date will move forward.

Reply:
right.